### DISCUSSION TITLE
Entities is Null

### DISCUSSION THREAD

**Discussion by ddoyley** [Xwilarg/TwitterSharp#13] [General]
I'm making the following call but Entities is returning null:-

`await twitterClient.GetTweetsFromUserIdAsync(userId, new[] { tweetOption.Entities }, null, null);`

I get the Id and Text property but everything else is null.

Am I doing something wrong?

**Comment by Xwilarg** [comment]
Uhm... It works well on my side
![image](https://user-images.githubusercontent.com/20053454/148703386-d6ea4209-7d25-4dde-aec2-51ea492a3409.png)
![image](https://user-images.githubusercontent.com/20053454/148703401-84b49e8f-d3a8-472f-af84-d5aaa4ee9582.png)

As you can see, I'm getting the hashtag properly

Here is my code if you want to reproduce the issue:
```cs
var client = new TwitterClient(Environment.GetEnvironmentVariable("TWITTER_TOKEN"));
var answer = await client.GetTweetsFromUserIdAsync("1109748792721432577", new TweetOption[] { TweetOption.Entities }, null, null);
var a = answer[0];
```

**Reply by ddoyley** [reply]
Thanks for your help. I notice the code above has `new TweetOption[] { TweetOption.Entities }`. The documentation just has `new[] { TweetOption.Entities }`.

However, I have tried your version and am still getting nulls.

**Comment by ddoyley** [comment]
Actually, no, now it's working! Thanks!

**Reply by Xwilarg** [reply]
Glad to hear it! What did you change for it to work?

**Comment by ddoyley** [comment]
I realised that not every tweet has the Entities property. I'm now using `TweetOption.Created_At` 
as that was the property I wanted anyway. I think that would be a better example for the documentation as well?

**Reply by Xwilarg** [reply]
Yup, my next plan is actually to write a better documentation than just the README

**Reply by Xwilarg** [reply]
Small update, documentation is now available at https://twittersharp.zirk.eu/ \
Still need to improve it a lot but at least it's here

**Reply by ddoyley** [reply]
That's great, thanks. If you want help with the docs let me know.

**Reply by Xwilarg** [reply]
If you want to contribute I would be happy to, however the documentation is generated from the comments in the code so I just need to improve them

**Comment by ddoyley** [comment]
Thanks.

I'm getting exceptions when I include `TweetOption.ReferencedTweets` but it's getting late now so will look at that tomorrow. Thanks for your help.

**Reply by Xwilarg** [reply]
Sure, feel free to give me your code and the exception you're getting so I can help with it!

**Reply by ddoyley** [reply]
The exception just says `invalid type` 
![image](https://user-images.githubusercontent.com/65024994/148771074-7b1e626b-e592-4829-8bfa-0d28957f707d.png)

**Reply by Xwilarg** [reply]
Can you please give me an user ID with which you get this exception?

**Reply by ddoyley** [reply]
Sure. 66673916

**Reply by Xwilarg** [reply]
The issue was indeed on my side, I'll upload a new version of the library to fix it

**Reply by Xwilarg** [reply]
The issue will be solved with version 1.1.1: https://www.nuget.org/packages/TwitterSharp/1.1.1

**Reply by ddoyley** [reply]
Thanks for the quick fix. I've just had a quick look and that seems to have addressed the issue for me as well.

**Reply by marcogruhl** [reply]
A little late, but i think it depends on this PR: #32 so this can be marked as answered.
